themes/default/content/docs/clouds/aws/get-started/deploy-changes.md

```diff
@@ -450,6 +449,12 @@ pulumi.export('bucket_endpoint', pulumi.Output.concat('http://', bucket.website_
 {{% choosable language go %}}
 
 ```go
+import (
+  "fmt"
+  "github.com/pulumi/pulumi-aws/sdk/v6/go/aws/s3"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
```

Comment:
Ah, good catch. I see with the other languages, we add these imports in the block just above this one. I wonder if we could do that there instead (e.g., as we do with the Java example)? 🤔

Reply:
Thank you, you are right, the stile doesn’t align with the rest of the languages. 🙂